tests/sparse_test.py

```diff
@@ -2549,6 +2549,19 @@ def test_bcoo_from_scipy_sparse(self, input_type, shape, dtype):
     M_bcoo = sparse.BCOO.from_scipy_sparse(M_sparse)
     self.assertArraysEqual(M, M_bcoo.todense())
 
+  @jtu.sample_product(
+    input_type=[scipy.sparse.coo_matrix, scipy.sparse.csr_matrix,
+                scipy.sparse.csc_matrix],
+    shape=[(5, 8), (8, 5), (5, 5), (8, 8)],
+    dtype=jtu.dtypes.floating + jtu.dtypes.complex,
+  )
+  def test_bcsr_from_scipy_sparse(self, input_type, shape, dtype):
```

Comment:
We could potentially merge this test with `test_bcoo_from_scipy_sparse` and call it `test_bcxx*`. Is that a good way to consolidate the tests?

Reply:
Yes, I'd say consolidate the tests. You can iterate over `cls=[sparse.BCOO, sparse.BCSR]` in the test parameterization

Reply:
Thanks! Done.